core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -132,7 +132,105 @@ describe('Overview', () => {
+                        jazzServices_po.getAPIGET().click();
+                        browser.sleep(5000);
+                        jazzServices_po.getTryOut().click();
+                        browser.sleep(5000);
+                        jazzServices_po.getStringA().sendKeys('Testing');
+                        jazzServices_po.getStringB().sendKeys('Jazz');
+                        browser.sleep(5000);
+                        jazzServices_po.getExecute().click();
+                        expect(jazzServices_po.serverResponse().getText()).toEqual('200');
+                        jazzServices_po.getAPIGET().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getAPIPOST()), timeOutHigh);
+                        expect(jazzServices_po.getAPIPOST().getText()).toEqual('POST');
+                        jazzServices_po.getAPIPOST().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getTryOut()), timeOutHigh);
+                        jazzServices_po.getTryOut().click();
+                        browser.sleep(2000);
+                        jazzServices_po.getExecute().click();
+                        browser.sleep(2000);
+                        expect(jazzServices_po.serverResponse().getText()).toEqual('200');
+                        browser.close();
+                  });
+                  browser.switchTo().window(handles[0]).then(function(){
+                        browser.sleep(10000);
+                        browser.wait(EC.visibilityOf(jazzServices_po.getMetrices()), timeOutHigh);
+                        jazzServices_po.getMetrices().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getXXError()), timeOutHigh);
+                        jazzServices_po.getXXError().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getXXErrorFive()), timeOutHigh);
+                        jazzServices_po.getXXErrorFive().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getCacheHitCount()), timeOutHigh);
+                        jazzServices_po.getCacheHitCount().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getCacheMissCount()), timeOutHigh);
+                        jazzServices_po.getCacheMissCount().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getCount()), timeOutHigh);
+                        jazzServices_po.getCount().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getIntegrationLatency()), timeOutHigh);
+                        jazzServices_po.getIntegrationLatency().click();
+                        browser.wait(EC.visibilityOf(jazzServices_po.getLatency()), timeOutHigh);
+                        jazzServices_po.getLatency().click();
+                  });
+            });
+            browser.sleep(10000);
+            jazzServices_po.getServiceFromAsset().click();            
```

Comment:
reasoning between line 206 and 207

Reply:
it requires here, because before moving to the next test case. we have to go from the beginning i mean we have to click the service again. this is also because metrics section again require some time to reflect the values.